client/packages/coldchain/src/Equipment/ImportAsset/UploadTab.tsx

```diff
@@ -30,21 +39,96 @@ interface ParsedAsset {
   [key: string]: string | undefined;
 }
 
-enum AssetColumn {
-  ASSET_NUMBER = 0,
-  CATALOGUE_ITEM_CODE = 1,
-  NOTES = 2,
-  SERIAL_NUMBER = 3,
-  INSTALLATION_DATE = 4,
-}
+const formatDate = (value: string): string | null =>
+  Formatter.naiveDate(DateUtils.getDateOrNull(value));
 
-// the row object indexes are returned in column order
-// which allows us to index the keys
-const getCell = (row: ParsedAsset, index: AssetColumn) => {
-  const rowKeys = Object.keys(row);
-  const key = rowKeys[index] ?? '';
-  return row[key] ?? '';
-};
+function getImportHelpers<T, P>(
```

Comment:
thinking that these can be lifted out when we generalise the import modal. was trying to make them generic